src/js/_enqueues/admin/common.js

```diff
@@ -1288,7 +1288,20 @@ $( function() {
 		dismissible = ( data.dismissible && data.dismissible === true ) ? ' is-dismissible' : '';
 		type        = ( data.type ) ? data.type : 'info';
 
-		$adminNotice = '<div id="' + data.id + '" class="notice notice-' + data.type + dismissible + '"><p>' + data.message + '</p></div>';
+		// Support for multiple message formats
+		var message = data.message;
+		if (data.successes || data.errors) {
+			message = 'Success: ' + data.successes + ' operations completed. Errors: ' + data.errors;
```

Comment:
This looks like it was here for testing; the message probably doesn't need to be emended.

Reply:
`		// Set the message directly from data.message
		var message = data.message;

		// Create the admin notice
		$adminNotice = '<div id="' + data.id + '" class="notice notice-' + type + dismissible + '"><p>' + message + '</p></div>';
` is this what you mean by  "the message probably doesn't need to be emended. "

Reply:
I mean that the line `message = 'Success: ' + data.successes + ' operations completed. Errors: ' + data.errors;` doesn't need to be there.